Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM amazonlinux:2017.03.1.20170812
+
+# lock yum to the same repository version
+RUN sed -i 's/releasever=.*/releasever=2017.03/g' /etc/yum.conf
```

Comment:
So that the build is repeatable, I locked the `releasever` to `2017.03` vs. latest. In particular, the issue was when building the `pycurl` library from source since that when you `yum install libcurl-devel` if it pulls latest, when the lambda function runs you get an error that the linked version of `libcurl` differs from the compiled version of `libcurl`.

To get around that, I just pinned the `yum` repository to that version.